src/pages/ModalPage.py

```diff
@@ -5,7 +5,7 @@
 SIGN_IN_BUTTON_TEXT = "SIGN IN"
 FORM_PAGE_LOGIN_TEXT = "LOGIN"
 FORM_PAGE_REGISTER_TEXT = "REGISTER"
-FORM_PAGE_BUTTON_SING_UP_TEXT = "Sign Up"
+FORM_PAGE_BUTTON_SING_UP_TEXT = "SIGN UP"
```

Comment:
Are you sure that we need to change the text?

Reply:
I received the error when I was trying to run test_landing_registration. Sign up button was not found because the text is "SIGN UP" not "Sign Up"